pyeudiw/trust/handler/interface.py

```diff
@@ -0,0 +1,15 @@
+from pyeudiw.trust.model.trust_source import TrustSourceData
+
+class TrustHandlerInterface:
```

Comment:
Could you please provide a brief desacription, comment or docstring on what is a `TrustHandlerInterface`, a `TrustParameterData` and a `TrustSourceData`?

IMO if authors are to create new implementations on `TrustHandlerInterface` in order to extend the tool capabilities (such as x509 and DID), they also need some information on what behaviour they must satisfy.

As is, I am personally confused as to what `extract(...)` and `verify(...)` should do - especially `verify` since it is presented as a static method with no arguments.

Reply:
Resolved [33c0a54](https://github.com/italia/eudi-wallet-it-python/pull/284/commits/33c0a54145e85bc387e4b4420245fc4af56abac8)